frontend/src/containers/RecipePage/Recipe.js

```diff
@@ -69,16 +88,7 @@ class Recipe extends Component {
             <Header user={user} setJwt={setJwt} />
           </div>
         </div>
-        <div>
-          <button className="btn navbar-btn btn-lg edit-button" type="button">
-            <img src="https://img.icons8.com/windows/32/ffa616/edit.png" alt="edit" />
-            <p id="edit-icon-text">Edit</p>
-          </button>
-          <button className="btn navbar-btn btn-lg cancel-button" type="button">
-            <img src="https://img.icons8.com/windows/32/ffa616/cancel.png" alt="delete" />
-            <p id="delete-icon-text">Delete</p>
-          </button>
-        </div>
+        { user ? renderButtons() : null }
```

Comment:
I think having a ternary here, and an if statement in the function is a little redundant. You can probably consolidate things by doing `user && user.id === userId` here and removing the if statement from your `renderButtons` function.